interface/forms/prior_auth/templates/prior_auth/general_new.html

```diff
@@ -1,64 +1,93 @@
+{**
+	* Prior auth form
+	*
+	* @package   OpenEMR
+	* @link      http://www.open-emr.org
+	* @author    Brady Miller <brady.g.miller@gmail.com>
+	* @copyright Copyright (c) 2017-2020 Brady Miller <brady.g.miller@gmail.com>
+	* @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+	*}
 <html>
 <head>
-{literal}
- <style>
-<!--
-td {
-	font-size:12pt;
-	font-family:helvetica;
-}
-li{
-	font-size:11pt;
-	font-family:helvetica;
-	margin-left: 15px;
-}
-a {
-	font-size:11pt;
-	font-family:helvetica;
-}
-.title {
-	font-family: sans-serif;
-	font-size: 12pt;
-	font-weight: bold;
-	text-decoration: none;
-	color: #000000;
-}
-
-.form_text{
-	font-family: sans-serif;
-	font-size: 9pt;
-	text-decoration: none;
-	color: #000000;
-}
+<title>{xlt t='Prior Authorization'|text}</title>
```

Comment:
remove the text call here (xlt already escapes)

Reply:
there's a lot of those `xlt t=` in the code base :|

Reply:
oh, the text after the pipe, sorry

Reply:
but now you know where the smarty xlt function is :)
(not easy to find custom smarty functions)